internals/src/read_uint.rs

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Provides functionality for reading a `usize` from an iterator of bytes.
+
+/// Reads a `usize` from an iterator.
+///
+/// `size` is the number of bytes to read, iterator is expected to return bytes in little-endian.
+///
+/// # Panics
+///
+/// If `size` is greater than 4 (we only need to read 32-bit integers).
```

Comment:
Stale.

Reply:
Fuck, you need a long stick you can hit me with when I do stupid shit like this.

Reply:
On my way to google where I can find a 14Mm stick.